K_Scaffold/Template v2/scaffold/tabs/_tabs.pug

```diff
@@ -33,27 +57,27 @@ mixin tabs(name="tabs",defaultActiveTabIndex=0)
       - {string} [tabName] - The name of the tab. Defaults to `tab` followed by a index count starting at 1 (so tab1, then tab2, ...)
       - {object} [button] - object passed to the +action mixin to build the action button for switching tabe. By default a trigger is defined, but you can override it.
       - {string} [container] - tag to use to contain the whole tab. A div by default but can be overridden.
-  mixin tab(tabName,button={},container="div")
+  mixin tab({tabName,button={},container="div"})
     - tabName = actionButtonName(replaceProblems(tabName || `tab${tabs.length + 1}`));
     - if (tabs.filter(tab => tab.name === tabName).length) { throw new Error(`Tab "${tabName}" already exists in "${name}".`); }
     //- Cleanup the name of the navigation button
     - button.name = `nav-tabs-${name}--${tabName}`;
     //- Cleanup the class, add our own internal classes
     - button.class = button.class ? replaceProblems(button.class) : "";
-    - button.class = `tabs__button tabs--${name}__button tabs--${name}__button--${tabName} ${button.class}`;
-    - button.class = button.class + (tabs.length === defaultActiveTabIndex ? " k-active-tab" : "");
+    - button.class = `tabs__button tabs--${name}__button tabs--${name}__button--${tabName}${!persistent && defaultActiveTab === tabName ? 'k-active-button' : ''} ${button.class}`;
     - button['data-container-button'] = name;
     - button['data-button'] = tabName;
+    - const content = button.content;
+    - delete button.content;
 
     //- If not provided, hook the button to the default tab switcher listener
-    - button.trigger = button.trigger || {triggeredFuncs:["_kSwitchTab"]};
+    - button.trigger = button.trigger || {triggeredFuncs:["kSwitchTab"]};
     //- Cleanup the class of the tab content passed through the implicit attributes
     //- and add our own internal classes
     - attributes.class = actionButtonName(replaceProblems(attributes.class || ""));
-    - attributes.class = `tabs__container tabs--${name}__container tabs--${name}__container--${tabName} ${attributes.class}`;
-    - attributes.class = attributes.class + (tabs.length === defaultActiveTabIndex ? " k-active-tab" : "");
+    - attributes.class = `tabs__container tabs--${name}__container tabs--${name}__container--${tabName}${!persistent && defaultActiveTab === tabName ? 'k-active-tab' : ''} ${attributes.class}`;
```

Comment:
Same as above: I think there is a missing space before `'k-active-tab'`, and the moveable class would be clearer in-browser when at the end
```javascript
- attributes.class = `tabs__container tabs--${name}__container tabs--${name}__container--${tabName} ${attributes.class}${!persistent && defaultActiveTab === tabName ? ' k-active-tab' : ''}`;
```

Reply:
Yep, same response